src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -7,10 +7,19 @@
 
 public class Navigate extends BasePageObject {
 
+  /**
+   * Adding default testing group is invoked explicitly in URL as temporary change until
+   * the end of experiment (01.07.2016) described in DAT-4483.
+   */
+
+  private static final String defaultTestingGroup =
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)